book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative('./rental')
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rental = Rental.new(date, self, person)
+    rentals << rental
```

Comment:
- Hey, you should only instantiate the class for now. Kindly remove this line as you'd do that later in the next milestone.